specification/sources/chapters/extensions/oppo/oppo_controller_interaction.adoc

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2023 OPPO
+//
+// SPDX-License-Identifier: CC-BY-4.0
+
+
+include::../meta/XR_OPPO_controller_interaction.adoc[]
+
+*Contributors*::
+    Haomiao Jiang, OPPO +
+    Buyi Xu, OPPO +
+    Yebao Cai, OPPO
+
+*Overview*
+
+This extension defines a new interaction profile for the OPPO Controller, including but not limited to OPPO VR Glasses Controller.
+
+*OPPO Controller interaction profile*
+
+Interaction profile path:
+
+* pathname:/interaction_profiles/oppo/controller
+
+Valid for user paths:
+
+* pathname:/user/hand/left
+* pathname:/user/hand/right
+
+This interaction profile represents the input sources and haptics on the OPPO Controller.
+
+Supported component paths:
+* On /user/hand/left only:
+** subpathname:/input/x/click
+** subpathname:/input/x/touch
+** subpathname:/input/y/click
+** subpathname:/input/y/touch
+** subpathname:/input/menu/click
+** subpathname:/input/heartrate/value
```

Comment:
The `heartrate` path needs to be decorated with `heartrate_oppo` as per https://registry.khronos.org/OpenXR/specs/1.0/html/xrspec.html#_adding_input_sources_via_extensions

Reply:
@Wallbraker Thanks for reviewing. I updated the path with _oppo decoration per your suggestion in a new commit in this pull request.
Feel free to let me know if there is anything else that needs to be updated.

Thanks